src/js/components/ServerErrorModal.js

```diff
@@ -56,7 +56,7 @@ module.exports = class ServerErrorModal extends mixin(StoreMixin) {
     });
   }
 
-  handleServerError(errorMessage) {
+  handleServerError({message:errorMessage}) {
```

Comment:
This isn't a good idea and will break our existing Server Errors. We should be using `RequestUtil.getErrorFromXHR(xhr)` in all the actions when the request errors. I just noticed we're using `RequestUtil.parseResponseBody(xhr)` in several places. https://github.com/dcos/dcos-ui/blob/master/src/js/events/MarathonActions.js#L112